internal/util/projutil/sdk_metrics_util.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package projutil
+
+import (
+	kbutil "github.com/operator-framework/operator-sdk/internal/util/kubebuilder"
+	ver "github.com/operator-framework/operator-sdk/version"
+)
+
+const (
+	operatorSDK = "operator-sdk"
+	mediatype   = "operators.operatorframework.io.metrics.mediatype.v1"
+	builder     = "operators.operatorframework.io.metrics.builder"
+	layout      = "operators.operatorframework.io.metrics.project_layout"
+)
+
+type Sdkmetrics struct {
+	SdkMetricsData map[string]string
+}
+
+func GenerateSDKMetrics() Sdkmetrics {
+	m := Sdkmetrics{
+		SdkMetricsData: map[string]string{
+			mediatype: getMediaType(),
+			builder:   getSDKBuilder(),
+			layout:    getSDKProjectLayout(),
+		},
+	}
+	return m
+
+}
+
+func getSDKBuilder() string {
+	builder := operatorSDK + "-" + ver.GitVersion
```

Comment:
This can be a `const`. Also
```suggestion
	builder := operatorSDK + "-" + ver.Version
```